### ISSUE TITLE
MAUI support

### ISSUE THREAD

**Issue by KazysNoobiys** [roubachof/Xamarin.Forms.Nuke#29]
Will maui be supported?

**Comment by LaurenC2022** [comment]
Yes, MAUI will be supported for 6 months after the new released version. 

> A major version of .NET MAUI will receive support for a minimum of 6 months after a successor (the next major release) ships. For example, .NET MAUI 6.0 will be supported for 6 months after .NET MAUI 7.0 ships. Similarly, .NET MAUI 7.0 will receive support for 6 months after .NET MAUI 8.0 ships.

Source: https://dotnet.microsoft.com/en-us/platform/support/policy/maui

**Comment by KazysNoobiys** [comment]
Um, there must be some kind of misunderstanding. I'm interested if Xamarin.Forms.Nuke MAUI will be supported?

**Comment by roubachof** [comment]
There will be a Maui.Nuke :)

**Comment by roubachof** [comment]
https://github.com/roubachof/Maui.Nuke
